src/ui/frmpreferences.cpp

```diff
@@ -92,6 +92,15 @@ void frmPreferences::on_treeWidget_currentItemChanged(QTreeWidgetItem *current,
 
 void frmPreferences::on_buttonBox_accepted()
 {
+    if(m_keyGrabber->findConflicts()){
+        QMessageBox msgBox;
+        msgBox.setWindowTitle(QCoreApplication::applicationName());
+        msgBox.setIcon(QMessageBox::Warning);
+        msgBox.setInformativeText(tr("Two or more actions share the same shortcut. These conflicts must be resolved before your changes can be saved."));
+        msgBox.exec();
+        return;
```

Comment:
It might be unwise to let them leave the preferences menu with 
conflicts, even optionally.... as it will break both of the shortcut 
keys in those cases due to how Qt works.  As a side note, auto-scrolling 
to the conflict would be a nice touch.

The original implementation was just to get conflict detection 
working... I never did get a chance to polish it very well. :)